src/Service.Host/client/src/components/PurchaseOrders/PurchaseOrderUtility.js

```diff
@@ -0,0 +1,767 @@
+                                value={`${order.requestedBy?.fullName} (${order.requestedBy?.id})`}
+                                label="Requested By"
+                                disabled
+                            />
+                        </Grid>
+                        <Grid item xs={12}>
+                            <InputField
+                                fullWidth
+                                value={`${order.enteredBy?.fullName} (${order.enteredBy?.id})`}
+                                label="Entered By"
+                                disabled
+                            />
+                        </Grid>
+                        <Grid item xs={3}>
+                            <Button
+                                className={classes.buttonMarginTop}
+                                color="primary"
+                                variant="contained"
+                                disabled={!allowedToAuthorise()}
+                                onClick={handleAuthorise}
+                            >
+                                Authorise
+                            </Button>
+                        </Grid>
+                        <Grid item xs={1}>
+                            <Tooltip title="Email to request authorisation">
+                                <IconButton
+                                    className={classes.buttonMarginTop}
+                                    aria-label="Email"
+                                    onClick={() => setAuthEmailDialogOpen(true)}
+                                    disabled={creating}
+                                >
+                                    <Email />
+                                </IconButton>
+                            </Tooltip>
+                        </Grid>
+                        <Grid item xs={8}>
+                            <InputField
+                                fullWidth
+                                value={
+                                    order.authorisedBy
+                                        ? `${order.authorisedBy?.fullName} (${order.authorisedBy?.id})`
+                                        : ''
+                                }
+                                label="Authorised by"
+                                disabled
+                            />
+                        </Grid>
+                        <Grid item xs={12}>
+                            <InputField
+                                fullWidth
+                                value={order.remarks}
+                                label="Remarks"
+                                propertyName="remarksForOrder"
+                                onChange={handleFieldChange}
+                                rows={4}
+                                disabled={!editingAllowed}
+                            />
+                        </Grid>
+                        {/* -------------- fields on details ------------------------- */}
```

Comment:
I think this comment is kinda redundant since the code on the next line sort of says the same thing